### PR TITLE
Closes #218: add support for pelican-toc plugin

### LINKED ISSUE #218: add support for pelican-toc plugin
I think it would be great if Flex can generate a table of content for each article, beacuse most technical blogs will be more easy to read with a table of content.

`pelican-toc` plugin is pretty good for me, I have made a little update in articlce.html to support it.

The live demo is here, ignoring the section number generated by `section_number` plugin.

http://guqian110.github.io/posts/python/pep8_in_practice.html

### PR DESCRIPTION
**Pull request by guqian110**
Closes #218

### CODE REVIEW

File: templates/article.html [hunk 1]
```diff
@@ -46,6 +46,12 @@ <h1 id="{{ article.slug }}">{{ article.title }}</h1>
     </script>
   {% endif %}
 
+  {% if article.toc %}
+  <div class="col-lg-3 hidden-xs hidden-sm">
```
Comment (alexandrevicenzi) [inline templates/article.html:50]: The theme is not bootstrap based, none of these classes will have effect.

**Comment by guqian110** [comment]
> Not sure how this looks like, but would be nice to make like [this TOC](https://mmistakes.github.io/minimal-mistakes/layout-table-of-contents-post/), on the side of the text, before the content only makes sense in mobile.

It looks like [this](http://guqian110.github.io/posts/python/pep8_in_practice.html). A toc on the side of text looks ugly to me, it would broke the minimalit style of theme. Maybe it's true that a toc is not necessary : -(

Should I close this PR?

**Comment by alexandrevicenzi** [comment]
TOC is useful, maybe it could be configured, like toc on the top, or on the right or no toc at all.

**Comment by guqian110** [comment]
> TOC is useful, maybe it could be configured, like toc on the top, or on the right or no toc at all.

I'm not a front-end programmer, I had read some articles to learn CSS and HTML in recent days and try to add some codes to the theme, but I failed to make it. Sorry for can't to contribute :-(

**Pull request closed without merging**